lib/Auth/Process/privacyidea.php

```diff
@@ -77,7 +91,7 @@ public function __construct(array $config, $reserved)
      */
     public function process(&$state)
     {
-
+	    SimpleSAML_Logger::info("privacyIDEA Auth Proc Filter: running process");
```

Comment:
Reading "running process" in a log file might be a bit misleading.
We should change this to something like "entering function 'process'".